common/autoware_test_utils/src/mock_data_parser.cpp

```diff
@@ -204,6 +204,17 @@ std::vector<LaneletSegment> parse(const YAML::Node & node)
   return segments;
 }
 
+template <>
+LaneletRoute parse(const YAML::Node & node)
```

Comment:
there's already lanelet route parser in the mock data parser, just that it has different input, so i think you should refactor the code to reuse this function.